addon/mixins/components/schema-field-initializer.js

```diff
@@ -36,6 +36,7 @@ export default Ember.Mixin.create({
 
       document.set(key, value);
       this.set('value', value);
+      this.sendAction('changed', value);
```

Comment:
In this case, this allows us to bind to the `changed` event of the component, so our consuming app can do something.

https://github.com/sandersonet/sheriff.aptible.com/blob/schema-api/app/components/security-control/template.hbs#L11

The above action is caught and ultimately triggers the "Save" button to show up.